CurrencyConverter/CurrencyConverter.Domain/Currency.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CurrencyConverter.Domain
+{
+    public class Currency
```

Comment:
What's the difference between this and CurrencyDto? Or since they look the same, where one is used and there another?

Reply:
Short answer, I'm not sure exactly why this is needed, but it has something to do with how API's should be designed and for API to be more reusable and loosely coupled.

https://en.wikipedia.org/wiki/Data_transfer_object